### PR TITLE
Tone back trust assumption in authn ceremony structures section

### PR DESCRIPTION
**Pull request by emlun**
This would merge into #956.

See https://github.com/w3c/webauthn/pull/842#discussion_r195887194 and https://github.com/w3c/webauthn/pull/899#discussion_r195171633

***
<a href="https://pr-preview.s3.amazonaws.com/emlun/webauthn/pull/960.html" title="Last updated on Jun 26, 2018, 3:42 PM GMT (40896e5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/960/5945017...emlun:40896e5.html" title="Last updated on Jun 26, 2018, 3:42 PM GMT (40896e5)">Diff</a>

### CODE REVIEW

**Comment by emlun** [comment]
This is in response to @equalsJeffH's review comment https://github.com/w3c/webauthn/pull/842#discussion_r195887194, which references previous discussion in https://github.com/w3c/webauthn/pull/899#discussion_r195171633. This would also merge into #956, not directly into master.

**Comment by emlun** [comment]
I opened this as a separate PR so that @equalsJeffH and I can easier iterate on it in isolation instead of pushing commits directly into #956.

File: index.bs [hunk 1]
```diff
@@ -2439,11 +2439,9 @@ private key=] are defined in [[#terminology]].
 The [=authentication ceremony structure=] is defined by the [=authentication factors=] used in the [=authentication
 ceremony|ceremony=]. Different [=authentication ceremony structures=] provide different benefits for security and usability.
 
-The following discussion assumes that the [=[RP]=] can verify the security guarantees of the [=authenticator=] by inspecting
-[=attestation statements=] generated by the [=authenticator=]. In particular, the [=[RP]=] cannot trust that the [=authenticator=]
-can verify [=something you know=] or [=something you are=] unless these properties can be derived from the [=attestation
-statement=]. Thus, an [=authenticator=] that cannot provide such an [=attestation statement=] can only enable the [=second-factor
-use case=]. See also [[#sctn-no-attestation-security-attestation]].
+The following discussion assumes that the [=[RP]=] to some degree trusts the [=authenticator=] to perform [=user verification=].
```
Comment (equalsJeffH) [inline index.bs]: suggest something like:

The following discussion is based on the notion that the degree of trust an [=[RP]=] has in an [=authenticator=] to satisfactorily perform [=user verification=] is established and assessed by the [=[RP]=].

Comment (emlun) [reply index.bs]: Looks good, thanks!

**Comment by emlun** [comment]
@selfissued Would you object to merging this into #956 (not directly into master)? The text whose deletion you asked about is [added][diff] in #956, and this cross-PR reformulates that paragraph.

[diff]: https://github.com/w3c/webauthn/pull/956/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR2467

**Comment by emlun** [comment]
I will merge this into #956 if I see no further objections before 2018-06-29T12:00Z.

**Pull request merged**